saleor/order/__init__.py

```diff
@@ -12,12 +12,14 @@ def ready(self):
 
 
 class OrderStatus:
+    DRAFT = 'draft'
     UNFULFILLED = 'unfulfilled'
     PARTIALLY_FULFILLED = 'partially fulfilled'
     FULFILLED = 'fulfilled'
     CANCELED = 'canceled'
 
     CHOICES = [
+        (DRAFT, pgettext_lazy('order status', 'Draft')),
         (UNFULFILLED, pgettext_lazy('order status', 'Unfulfilled')),
```

Comment:
Should we profit from the fact that we are editing the order process, to edit contexts to more descriptive contexts? Explaining to the translators what triggers each status.

Reply:
This is a clever idea, maybe the statuses' contexts could be more descriptive.